### DISCUSSION TITLE
Monitor ingest rates

### DISCUSSION THREAD

**Discussion by BishalLakha** [Security-Onion-Solutions/securityonion#6803] [General]
I am trying to monitor data size and ingest rate through kibana like they mentioned in the video below. Have anyone  integrated mapper-size plugin in security onion ? And are there any other tools that could be used to monitor this?
https://www.youtube.com/watch?v=CnwnaB7OMIA&t=1162s

**Comment by TOoSmOotH** [comment]
Have you looked at the pipeline graphs in grafana?
